### PR TITLE
christina-roperto/c04-iam01

### PR DESCRIPTION
**Pull request by christina-roperto**
# Description

Exercise c04-iam01

Student Name: Christina Roperto

>creating a new user with read only access to S3 (only devops-iac01 bucket as I wanted it to be as specific as possible). Let me know if you want me to loosen the resource 

## Type of change

- [x] Exercise Submission (one exercise per PR)
  - [x] This PR is just for **one** exercise of a class
  - [x] PR name follows the pattern `<github-username>/<exercise-number>`
  - [x] Added exercise files inside folder `/classes/<class- number>/exercises/<exercise-number>/<github-username>/`
	- [x] Review [exercise submission steps](./README.md#Exercises)
- [ ] Class content (instructors/admins)
- [ ] Documentation update
- [ ] Repo management

### CODE REVIEW

**Comment by denstorti** [comment]
Hi Christina, have you been pulling frequently from `master` before creating a new dev branch?

I see you are using your forked repo instead of repo `devopsacademyau/academy` to create your branches and PRs. There is no problem doing that, but you need to sync your forked repo with the original frequently.

I can see your `master` in https://github.com/christina-roperto/academy has 218 commits.
While master in `devopsacademyau/academy` has 416 commits. 

Let me know if you need help with the steps to synchronize them.

**Comment by christina-roperto** [comment]
Thanks for the help Denis. I've decided to just ignore my forked master and track the devops academy. Is it ok?

File: classes/04class/exercises/c04-iam01/christina-roperto/S3readOnlyPolicy.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Effect": "Allow",
+            "Action": [
+                "s3:GetObject",
```
Comment (caiocezart) [inline classes/04class/exercises/c04-iam01/christina-roperto/S3readOnlyPolicy.json:7]: your answer is not wrong but if your bucket has `versioning` enabled (if you update a file, it keeps both versions as history) you won't be able to get the object as you would need the `s3:GetObjectVersion` as well, hence why the exercise requests `s3:GetObject*`

Comment (christina-roperto) [reply classes/04class/exercises/c04-iam01/christina-roperto/S3readOnlyPolicy.json]: Hi Caio, thanks for letting me know this. After a bit of checking the help list and googling, I realised that it's not so easy to update/edit policy via cli so I update via console 😬let me know if it's not ok!

**Pull request merged**